proposals/0000-a11y-cross-platform.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Cross-platform Accessibility Improvements
+author:
+- Marc Mulcahy
+date: 2018-10-31
+---
+
+# RFC0000: Cross-platform Accessibility Improvements
+
+## Summary
+
+Several accessibility-related properties are only available on one platform (either iOS or Android, but not both). This proposal attempts to make as many properties available on both Android and iOS as possible.
+
+Properties only available on iOS include:
+
+* accessibilityViewIsModal
+* accessibilityElementsHidden
+
+Properties only available on Android include:
+
+* accessibilityLiveRegion
+
+Several properties are missing from both platforms:
+
+* accessibilityCompoundComponent: replace accessible-- indicates to assistive technologies that all the components inside this one should be treated as a single component.
+* accessibilityRelationships: describes how this component is related to others
```

Comment:
I have a proposal internally that is based on Web's ARIA properties: `accessibilityLabelID` (equivalent to `aria-labelledby`), `accessibilityDescriptionID` (equivalent to `aria-describedby`), and `accessibilityFocusID` (equivalent to `aria-activedescendent`). All these props would take the value of the `nativeID` of the element they are associated with.

Reply:
Makes sense. From naming point of view, are you in favor of using the ARIA namespace, or creating our own properties based on ARIA?

Reply:
What do you mean by "ARIA namespace"? Something like `accessibilityLabelledBy` vs `accessibilityLabelID`?